frontend/views/containers/chatroom/MessageReactions.vue

```diff
@@ -82,6 +82,7 @@ export default ({
 .c-emoticons-list {
   padding-left: 3rem;
   display: flex;
+  flex-wrap: wrap;
```

Comment:
This is a fix for the overflowing emoji issues @Silver-IT  found.

<img src='https://github.com/user-attachments/assets/97a97996-bfd2-480c-80df-7532a0f580ae' width='430'>